src/windows.rs

```diff
@@ -25,25 +22,18 @@ pub fn home_dir_inner() -> Option<PathBuf> {
 #[cfg(not(target_vendor = "uwp"))]
 fn home_dir_crt() -> Option<PathBuf> {
     unsafe {
-        let me = GetCurrentProcess();
-        let mut token = ptr::null_mut();
-        if OpenProcessToken(me, TOKEN_READ, &mut token) == 0 {
-            return None;
+        let mut path: [MaybeUninit<u16>; MAX_PATH] = MaybeUninit::uninit().assume_init();
```

Comment:
All this `MaybeUninit` stuff seems a bit too much. I favor the much simpler `let mut path = [0u16; MAX_PATH];`.

Reply:
Thanks for your attention. But doesn't that generate memset?

Reply:
Thanks for your attention. But doesn't that generate memset?

Reply:
I think I will use `Vec::with_capacity` which skip the complexity of MaybeUninit.

Reply:
Thank you for taking time to review.